DirectEmployers/MyJobs
# PD-2737, modify logo to match Prod, page elements re-alignment

Purpose of PR: 
     *Re-draw the svg dot jobs logo to match Production, but change the dot to our light blue color.
     *Align the top of the dot jobs logo to be on the same horizontal plane as the top of the input boxes.  Left align the dot jobs logo with the facets on the vertical axis.
     *Horizontally align the "Found 123... Jobs" text with the "Network ..yadayada" text.
     *Removed the share icon button, added social media icons below the "Show More Jobs" button. Changed the "Search" button text to "Search Jobs".
     *Started the process of using html5 semantics tags in our templates, which was one of our directive from a couple of months ago.

To Test:
     1. Please switch to v2 template.
     2. Please ensure alignments are as described above (pics below).
     3. Please click the Search Jobs button without any value in the input boxes, please ensure the Found 123... Jobs text stays in alignment with the Network text in various desktop screen sizes.
     4. Please enter value in input boxes so that bread crumbs shows up once jobs are returned, please ensure Found Jobs text stays in alignment with Network text.
     5. Please ensure the Social Media icons appears below and to the right of the Show More Jobs button.
     6. Mobile view remains the same.
Before:
![before](https://cloud.githubusercontent.com/assets/5124153/20673280/7e585f3e-b552-11e6-91e5-83c046cf1170.png)

After:
![after1](https://cloud.githubusercontent.com/assets/5124153/20673286/86cee426-b552-11e6-8b30-950be46e36d1.png)

After (social media share):
![after2](https://cloud.githubusercontent.com/assets/5124153/20673295/8b0d9618-b552-11e6-962d-7bfd42e13267.png)
